datafusion/sqllogictest/test_files/subquery.slt

```diff
@@ -1080,8 +1080,8 @@ query TT
 explain select a/2, a/2 + 1 from t
 ----
 logical_plan
-01)Projection: {t.a / Int64(2)|{Int64(2)}|{t.a}} AS t.a / Int64(2), {t.a / Int64(2)|{Int64(2)}|{t.a}} AS t.a / Int64(2) + Int64(1)
-02)--Projection: t.a / Int64(2) AS {t.a / Int64(2)|{Int64(2)}|{t.a}}
+01)Projection: #1 AS t.a / Int64(2), #1 AS t.a / Int64(2) + Int64(1)
```

Comment:
Second projection is `#1 + 1` , but the alias makes it hard to read. Can't figure out how to fix this, appreciate any feedback.

Reply:
But don't we need to keep the `AS t.a / Int64(2)` alias in this case so as to keep the original top level column name?

Reply:
It must be kept for the first projection, yes.

Ideally, the second projection IMO should be `#1 + 1 AS ...`.